src/javascript/app/base/binary_loader.js

```diff
@@ -78,8 +92,13 @@ const BinaryLoader = (() => {
         ContentVisibility.init().then(() => {
             BinarySocket.wait('authorize', 'website_status', 'landing_company').then(() => {
                 GTM.pushDataLayer({ event: 'page_load' }); // we need website_status.clients_country
-                LiveChat.init();
-
+                if (!window.LiveChatWidget){
```

Comment:
i think we can initialize after destroy, and add check in the livechat file itself for this rather than here for better specificity